### ISSUE TITLE
Bottom Navigation is not available on android

### ISSUE THREAD

**Issue by susaurabh-git** [react-native-training/react-native-elements-app#233]
@arpitBhalla  When I ran the demo app on android device and emulator, I don't see the bottom navigation, please look into this
<img width="371" alt="Screen Shot 2021-09-19 at 10 37 27 PM" src="https://user-images.githubusercontent.com/20015708/133952446-49b35d7a-4997-4432-8e45-e315713d9bcb.png">
.

**Comment by arpitBhalla** [comment]
There is no bottom navigation 🙄

**Comment by susaurabh-git** [comment]
@arpitBhalla - Interestingly, I saw the bottom navigation on "https://react-native-elements.github.io/react-native-elements-app " as well as on the readme page of this application.

<img width="1271" alt="Screen Shot 2021-09-20 at 9 22 08 PM" src="https://user-images.githubusercontent.com/20015708/134098185-6d438376-b5f0-45ac-b23b-25d1f32584de.png">

<img width="251" alt="Screen Shot 2021-09-20 at 9 22 33 PM" src="https://user-images.githubusercontent.com/20015708/134098190-ada6100a-9ebb-4f64-8ed4-213ec17bbdab.png">

<img width="272" alt="Screen Shot 2021-09-20 at 9 23 59 PM" src="https://user-images.githubusercontent.com/20015708/134098308-d9b53d7f-2077-4aea-9480-91bee15635d7.png">

**Comment by susaurabh-git** [comment]
@arpitBhalla you are correct, I checked the code. there is no bottom tab. I tried using '@react-navigation/bottom-tabs' but it is conflicting. I am getting the following error
<img width="502" alt="Screen Shot 2021-10-01 at 12 08 50 PM" src="https://user-images.githubusercontent.com/20015708/135652913-deedf732-296a-48ec-9535-d52e9a179781.png">
**dependencies:**
`    "@react-navigation/bottom-tabs": "^5.11.15",
    "@react-navigation/drawer": "^5.12.9",
    "@react-navigation/native": "^5.9.8",
    "@react-navigation/stack": "^5.14.9",`
any suggestion here
[RootNavigator-tsx.txt](https://github.com/react-native-elements/react-native-elements-app/files/7268630/RootNavigator-tsx.txt)
